data/i18n/en/en.toml

```diff
@@ -519,12 +519,27 @@ other = "2006-01-02"
 [release_date_format_month]
 other = "January 2006"
 
+# Use 2006-01-02 (ISO 8601) if you are not sure what to use.
+[release_date_format_text]
+other = "Monday, 02 Jan 2006"
+
 [release_cherry_pick_deadline]
 other = "Cherry Pick Deadline"
 
 [release_end_of_life_date]
 other = "End Of Life Date"
 
+# Also see release_maintenance_and_end_of_life_details_past
+[release_maintenance_and_end_of_life_details_current]
+other = "Kubernetes {{ .minor_version }} enters maintenance mode on {{ .maintenance_mode_start_date }}; the End of Life date for Kubernetes {{ .minor_version }} is {{ .release_eol_date }}."
```

Comment:
Not sure if this will work.
I'm not aware of any convention related to the order of template variable substitution and the replacement of localized strings. This uncertainty could be the reason why we have partial sentences in the current file like "xxx_before" or "xxx_1" and "xxx_after".

Reply:
It does work; check the page previews to see how it looks.

> partial sentences in the current file like "xxx_before" or "xxx_1" and "xxx_after".

The approach I'm switching to here is what Hugo itself recommends; what we have is more of a bodge (I can say that, because some of those bodges are by me from before I realized the right way).